packages/assets/README.md

```diff
@@ -1,28 +1,37 @@
-# `@atb-as/assets`
+# `@atb-as/generate-assets`
 
 ## Usage
 
 ### Run as CLI:
 
 Format:
 
-```sh
-usage: npx @atb-as/generate-assets <orgId> <output directory> [--glob <string>]
+```
+Usage: npx @atb-as/generate-assets [options] <type> <orgId>
 
-Outputs assets for a specific organization in the specified output directory.
+Arguments:
+  type                    Type of assets to generate (choices: "colors", "all", "mono")
+  orgId                   Generate for specific organization (choices: "atb", "nfk")
 
-  Example: npx @atb-as/generate-assets atb ./static --glob "**.svg"
+Options:
+  -o, --out-dir <output>  Output directory
+  -d, --debug             Log all files generated (default: false)
+  -im, --ignore-mono      Ignore generating mono-icons with theme colors. (default: false)
```

Comment:
Hva er egentlig bruksområdet til dette flagget nå etter at man kan spesifisere type?

Reply:
Tror kanskje flagget burde være mer tydelig i navn eller at beskrivelsen min er dårlig 😅 . Hensikten for flagget er å unnlate å gjøre faktisk generering av to ulike mono-typer for dark/light mode. For Webshop vil du gjerne ha de generert f.eks, men for App så skjer dette som en del av ThemeText og bruk av SVGR til å fjerne fargene. Så da er de genererte bildene (dark/light) bare ekstravekt til bundle.